lvis/eval.py

```diff
@@ -39,6 +42,21 @@ def __init__(self, lvis_gt, lvis_dt, iou_type="segm"):
         else:
             raise TypeError("Unsupported type {} of lvis_dt.".format(lvis_dt))
 
+        # Precompute boundary.
+        if self.use_boundary_iou:
+            if not self.lvis_gt.get_boundary:
```

Comment:
How about renaming `LVIS` and `LVISResults` `get_boundary` to `needs_boundary`? That seems clearer to me (`get_*` sounds like a function name).

Reply:
I actually thought about naming it as `precompute_boundary`, but I kept it as `get_boundary` to be consistent with the Boundary IoU API.

Do you think `precompute_boundary` is better than `needs_boundary`?

Reply:
I think `precompute_boundary` is also good.

Reply:
Sounds good, I renamed it to `precompute_boundary`